sklearn/tests/test_multioutput.py

```diff
@@ -386,17 +386,27 @@ def test_multi_output_exceptions():
     # NotFittedError when fit is not done but score, predict and
     # and predict_proba are called
     moc = MultiOutputClassifier(LinearSVC(random_state=0))
-    assert_raises(NotFittedError, moc.predict, y)
+
+    with pytest.raises(NotFittedError):
+        moc.predict(y)
+
     with pytest.raises(NotFittedError):
         moc.predict_proba
-    assert_raises(NotFittedError, moc.score, X, y)
+
+    with pytest.raises(NotFittedError):
+        moc.score(X, y)
+
     # ValueError when number of outputs is different
     # for fit and score
     y_new = np.column_stack((y1, y2))
     moc.fit(X, y)
-    assert_raises(ValueError, moc.score, X, y_new)
+    with pytest.raises(ValueError):
```

Comment:
Could you please add a check on the message itself for this one. Then you can probably remove the inline comment above because it's going to be redundant with the check itself.

Reply:
Pushed new changes to amend this. Thanks @ogrisel.